### DISCUSSION TITLE
Unable to use ngx-toaster package in remote which is working in host.

### DISCUSSION THREAD

**Discussion by ShussainML** [nrwl/nx#22825] [General]
I have installed ngx-toaster, and use it one of component of host , and its working, but when I am trying to use it remote component, its showing error. I am using dynamic federation.
remote app config
`provideAnimations(),
    provideToastr({
      timeOut: 10000,
      positionClass: 'toast-top-right',
      preventDuplicates: true,
      progressAnimation:'decreasing',
      closeButton:true
    }),`
    remote app config
    ` provideAnimations(),
     provideToastr(),`
     getting this
     
![image](https://github.com/nrwl/nx/assets/83750648/64f561e8-de38-41a7-83aa-34759925a25a)
also used this approach

**Comment by l-Legacy-l** [comment]
I'm having the same issue, any help ?

**Comment by hoan02** [comment]
same issue
